assets/js/modal.js

```diff
@@ -4,4 +4,14 @@
 
 function closeGreeting(){
     document.querySelector('.greetings').classList.toggle('close');
+}
+
+function playAgain() {
+    closeGreeting();
+    handlePlayPress(document);
```

Comment:
change `handlePlayPress(document);` with `handlePlayPress(document.querySelector('#play'));`

**explanation**
As you can see in `morseSound.js` file, there `handlePlayPress()` is defined. In that, on line 51:-
`code = obj.querySelector('.answer').textContent;` is written. 

using `handlePlayPress(document);` completely works fine, because currently there is only one element with `class` name `answer`. so fortunately, this line `code = obj.querySelector('.answer').textContent;` finds our desired element .

But consider a case when there is another element with class name `answer`. There, the problem might arise of selecting the wrong element with `answer` class.

Thus using `handlePlayPress(document.querySelector('#play'))` will precisely select our desired  element with class name `answer`. Note that element with id `play` is unique and cant be made twice, therefore, using this element is safer.